browser/src/layer/tile/CommentListSection.ts

```diff
@@ -234,8 +234,8 @@ export class CommentSection extends CanvasSectionObject {
 		var openArray = [];
 
 		for (var i = 0; i < this.sectionProperties.commentList.length; i++) {
-			openArray.push(this.sectionProperties.commentList[i]);
-			if (this.sectionProperties.commentList[i].sectionProperties.data.parentId === '0') {
+			if (this.sectionProperties.commentList[i].sectionProperties.data.parent === '0') {
```

Comment:
please add human readable method wit that check:    `if (this.sectionProperties.commentList[i].isSomething())` to remove magic number